### PR TITLE
chore(.storybook)configure-storybook-187364803

### PR DESCRIPTION
**Pull request by shebz2023**
**what does this Pr do?**

this pr configures storybook,it provides a sandbox environment for _developing, testing, and documenting UI components_ in isolation

**how can this be tested?**

**RUN**          _npm run storybook_

**relevant pivotal tracker id**
#187364803
Screenshot 2024-06-12 112727

### CODE REVIEW

**Review by martinemahirwe — APPROVED** [review]
Good work

**Review by Sagastein — APPROVED** [review]
LGTM

File: .storybook/main.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "@storybook/addon-onboarding",
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@chromatic-com/storybook",
+    "@storybook/addon-interactions",
+  ],
+  framework: {
+    name: "@storybook/react-vite",
+    options: {},
+  },
```
Comment (SergeInGithub) [inline .storybook/main.ts:24]:
```suggestion
  },
  docs: {
    autodocs: 'tag',
  },
```

File: .storybook/main.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  addons: [
+    "@storybook/addon-onboarding",
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@chromatic-com/storybook",
+    "@storybook/addon-interactions",
+  ],
+  framework: {
+    name: "@storybook/react-vite",
+    options: {},
```
Comment (SergeInGithub) [inline .storybook/main.ts]:
```suggestion
    options: {
      fastRefresh: true,
    },
```

File: .storybook/preview.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import type { Preview } from "@storybook/react";
+
```
Comment (SergeInGithub) [inline .storybook/preview.ts]: import our css file here as well. Will come in handy later on
```suggestion
import '../src/index.css';

```

**Review by SergeInGithub — COMMENTED** [review]
Looks good so far. Look into some suggestions below though @shebz2023

**Review by SergeInGithub — COMMENTED** [review]
Nice work, just rebase @shebz2023

**Review by SergeInGithub — APPROVED** [review]
LGTM

**Pull request merged**